ui/js/api.js

```diff
@@ -107,7 +107,8 @@ function get_members() {
                 .css('max-height', '500px')
                 .css('overflow-y', 'auto');
             $.each(json.data, function(idx, member){
-                html.append(member.name + "<br />");
+                var mailto = '<a href="mailto:' + member.email + '">' + member.name + '</a>';
```

Comment:
The email field is optional and is not guaranteed to be present. Will this throw an exception if it's not there?